src/containers/Admin/EditTab/index.tsx

```diff
@@ -184,11 +182,41 @@ const EditTab = (): JSX.Element => {
         setSettings,
     ])
 
+    //burde denne flyttes ut...(?)
+    const GET_STOPPLACES = gql`
+        query getStopPlaces {
+            stations(lat: 59.911491, lon: 10.757933, range: 500, count: 25) {
```

Comment:
enig i flytting, not sure where tho, + legge lat lon range etc. som variabler

Reply:
jeg bare brukte den funksjonen som jeg la til i logic mappen. Jeg hadde jo allerede laget den (den jeg brukte de andre stedene)